rules/cloud/azure/azure_app_highly_privileged_permissions.yml

```diff
@@ -0,0 +1,19 @@
+title: App Granted Highly Privileged Permissions
+id: 5f7b4f82-f086-4bc3-b1d7-032f1264c648
+description: Detects when an app is granted broad permissions such as ".All" (Directory.ReadWrite.All) or other wide ranging permissions (Mail.).
+author: Bailey Bercik '@baileybercik', Mark Morowczynski '@markmorow'
+date: 2022/06/30
+references: https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/security-operations-applications#application-granted-highly-privileged-permissions
+logsource:
+    product: azure
+    service: auditlogs
+detection:
+    selection:
+        properties.message:
+            - Add app role assignment to service principal
```

Comment:
As there is only 1 field message no need a list
`properties.message:  Add app role assignment to service principal`